exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillRuleSets.java

```diff
@@ -133,7 +133,7 @@ public static RuleSet getDrillUserConfigurableLogicalRules(OptimizerRulesContext
       DrillPushProjectPastFilterRule.INSTANCE,
       DrillPushProjectPastJoinRule.INSTANCE,
       DrillPushProjIntoScan.INSTANCE,
-      DrillProjectSetOpTransposeRule.INSTANCE,
+      //DrillProjectSetOpTransposeRule.INSTANCE,
```

Comment:
Pls file a JIRA for re-enabling the rule in a subsequent release.  Also add a brief comment here referencing DRILL-3257